cmd/boulder-ra/main.go

```diff
@@ -206,10 +208,28 @@ func main() {
 	err = rai.UpdateIssuedCountForever()
 	cmd.FailOnError(err, "Updating total issuance count")
 
+	var grpcSrv *grpc.Server
+	if c.RA.GRPC != nil {
+		s, l, err := bgrpc.NewServer(c.RA.GRPC, scope)
```

Comment:
Let's use longer variable names here since it's not clear what these are. Also, can we use `grpcSrv` here in place of s.